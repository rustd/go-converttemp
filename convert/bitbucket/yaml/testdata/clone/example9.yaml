pipelines:
  default:
    - step:
        name: Clone with lfs on
        clone:
          lfs: true
        script:
          - ls -lh large-file.zip # 26M large-file.zip
    - step:
        name: Clone with lfs off
        clone:
          lfs: false
        script:
          - apt-get update && apt-get install -y git-lfs
          # Download only desired files
          - git lfs pull --include=large-file.zip
          - ls -lh large-file.zip # 26M large-file.zip
