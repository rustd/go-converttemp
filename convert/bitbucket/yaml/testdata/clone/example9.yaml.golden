{
    "pipelines": {
        "default": [
            {
                "step": {
                    "name": "Clone with lfs on",
                    "clone": {
                        "lfs": true
                    },
                    "script": [
                        "ls -lh large-file.zip"
                    ]
                }
            },
            {
                "step": {
                    "name": "Clone with lfs off",
                    "clone": {},
                    "script": [
                        "apt-get update && apt-get install -y git-lfs",
                        "git lfs pull --include=large-file.zip",
                        "ls -lh large-file.zip"
                    ]
                }
            }
        ]
    }
}